docs/markdown/glossary.md

```diff
@@ -0,0 +1,260 @@
+# Glossary
+
+This glossary provides definitions for key terms in the Interledger ecosystem.
+
+## Account
+   
+An accounting relationship between two Interledger participants. In Interledger, two participants establish an account with one another to track the current obligations they hold with one another.                                                                         
+
+## Address
+
+See instead: [Interledger Address](#interledger-address)
+
+## Application Layer
+
+The set of protocols built upon the Transport Layer that communicate payment details between senders and receivers. 
+
+Application Layer protocols specify all of the data and methods required to set up a payment for a specific category of use cases or applications. For example, the Simple Payment Setup Protocol uses WebFinger and HTTPS for account detail lookup.                                                                       
+
+## Arbitrage
+
+The practice of doing cyclic transactions which start and end on the same ledger, or groups of simultaneous transactions which add up to one cyclic transaction. The goal is to achieve a destination amount which is higher than the source amount by taking advantage of fluctuations in exchange rates.                                                                       
+
+## Asset
+
+Something you can own that is fungible. Can be a physical item; a quantity of a physical substance; a specific physical collection of coins or banknotes; a non-physical quantity of currency; a non-physical claim to a resource; a reputation for keeping one's word; someone else's intention to obey the terms of a contract; etc. Some types of assets can be more easily traded on a ledger than others. For some asset pairs it is easier to determine a fair exchange rate than for others.                                                                       
+
+## Balance 
+
+An account balance is the net total of the amounts on any packets "successfully" routed between the peers. When a node sends an ILP Prepare packet to a peer, and receives a valid ILP Fulfill response, before the expiry of the ILP Prepare, their balance with that peer decreases. When a node receives an ILP Prepare packet and returns a valid ILP Fulfill response, before the expiry of the ILP Prepare, their balance with that peer increases.                                                                        
+
+## Bandwidth
+
+The limit, set by a router, to the total value of ILP packets an account can send during a given time period. The limit may either be based on how much the router trusts the account-holder, or it may be used to prevent one account from tying up all of the router's bandwidth with its peers.                                                                       
```

Comment:
I'm a bit confused.  What's the difference between throughput and bandwidth?

Reply:
Great discussion. It seems like we should stop using the term "bandwidth" to describe `max-in-flight`. Reason being is that bandwidth is typically measured relative to some frequency. E.g., Internet Pipe Bandwidth: 5gb-per-second; Water Pipe Bandwidth: 3 liters-per-second; RF Antenna: 30 GHz; etc. 

However, "max in flight" doesn't necessarily correlate to a frequency or period. If my `max-in-flight` is 5, then it doesn't matter if I have 5 units in-flight for 1 minute or 1 hour.

If this is correct, then I would expect something like the following definitions to hold:

* **Account Max In-Flight**: The maximum value an account can have in-flight at any given moment in time (e.g., $5). 
* **Account Bandwidth**: The maximum value that can be streamed per-second ($5 / second).
* **Account Throughput**: The current value being transferred over a given time period. (e.g., Your _account bandwidth_ is currently $5/second, but your _account throughput_ is only $1/second because maybe the receiver can only receive that much per second for some reason).

All that to say, maybe the definition in the current PR should just say this:

```suggestion
The limit, set by a router, of the total value an account can have pending, or "in-fligh" at any given moment in time. The limit may either be based on how much the router trusts the account-holder, or it may be used to prevent one account from tying up all of the router's liquidity with its peers.
```

Reply:
Updated the definition with suggested text by @sappenin 
https://github.com/interledger/interledger.github.io/pull/123/files#diff-0f4b6322d1459dbe5c7f5eb93d2cd2efR33